templates/nginx.conf.j2

```diff
@@ -0,0 +1,34 @@
+user  nginx;
+worker_processes  auto;
+worker_rlimit_nofile {{ nginx_rlimit_nofile }};
+
+error_log  /var/log/nginx/error.log warn;
```

Comment:
maybe `error_log stderr warn;` to let journalctl handle the rest? Or, maybe you are addressing this already with symlinks or sth.

Reply:
:+1: